### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with Page Titles

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4909]
# Introduction
This is a site-wide accessibility issue that I found.

## Issue
Page titles are not reflective of the page that a user is in. Regardless of what page a user is in, the title that shows in the browser is always the same. This is a violation of WCAG Success Criterion 2.4.2 (Level A) - Page Titled. [See](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)

## Description
Success Criterion 2.4.2 (Level A) - [Web pages](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html#dfn-web-page) have titles that describe topic or purpose.

## Impact
- This criterion benefits all users in allowing users to quickly and easily identify whether the information contained in the Web page is relevant to their needs.
- People with visual disabilities will benefit from being able to differentiate content when multiple Web pages are open.
- People with cognitive disabilities, limited short-term memory and reading disabilities also benefit from the ability to identify content by its title.
- This criterion also benefits people with severe mobility impairments whose mode of operation relies on audio when navigating between Web pages.

## Fix
I will link a Pull Request to solve this problem.
